cloudsim-plus-testbeds/src/main/java/org/cloudsimplus/testbeds/ExperimentRunner.java

```diff
@@ -417,7 +423,11 @@ public ContinuousDistribution createRandomGenAndAddSeedToList(int experimentInde
      * generates uniform values between [0 and 1[. Adds the PRNG seed to the
      * {@link #getSeeds()} list. If it is to apply the
      * {@link #isApplyAntitheticVariatesTechnique() "Antithetic Variates Technique"}
+<<<<<<< HEAD
```

Comment:
Remove the <<<<<<<<< HEAD and leave the line after the =========, removing the final >>>>>> upstream/master line